network/junos/junos_config.py

```diff
@@ -77,9 +80,12 @@
     description:
       - The C(replace) argument will instruct the remote device to
         replace the current configuration hierarchy with the one specified
-        in the corresponding hierarchy of the source configuraiton loaded
+        in the corresponding hierarchy of the source configuration loaded
         from this module.
     required: true
+    choices:
```

Comment:
We normally don't add 'choices' to boolean arguments, just state that it is a boolean.

YAML itself uses a list ([y, yes, on, 1, true] and case variants)  to convert into True/False, we try to match that in jinja2 and ansible module processing.